modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentConfiguration.java

```diff
@@ -500,86 +500,85 @@ public void load(URL cfgUrl) throws IOException {
             props.load(reader);
         }
 
-        String val = props.getProperty("tokens");
+        List<String> items = split(props.getProperty("tokens"));
 
-        if (val != null)
-            tokens(new ArrayList<>(Arrays.asList(val.split(","))));
+        if (!F.isEmpty(items))
+            tokens(new ArrayList<>(items)); // Intentionally wrapped by ArrayList, for further manipulations.
 
-        val = props.getProperty("server-uri");
+        String val = props.getProperty("server-uri");
 
-        if (val != null)
+        if (!F.isEmpty(val))
             serverUri(val);
 
-        val = props.getProperty("node-uri");
+        items = split(props.getProperty("node-uri"));
 
-        // Intentionally wrapped by ArrayList, for further manipulations.
-        if (val != null)
-            nodeURIs(new ArrayList<>(Arrays.asList(val.split(","))));
+        if (!F.isEmpty(items))
+            nodeURIs(new ArrayList<>(items)); // Intentionally wrapped by ArrayList, for further manipulations.
```

Comment:
All calls of `split` is wrapped by `new ArrayList<>`, move this into `split` function

Reply:
Fixed.